packages/cashier/src/components/cashier-locked/__tests__/cashier-locked.spec.tsx

```diff
@@ -1,26 +1,29 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
 import CashierLocked from '../cashier-locked';
-
-jest.mock('Stores/connect', () => ({
-    __esModule: true,
-    default: 'mockedDefaultExport',
-    connect: () => Component => Component,
-}));
+import { StoreProvider } from '../../../hooks';
+import { TRootStore, DeepPartial } from '../../../types';
 
 describe('<CashierLocked />', () => {
     it('should show the proper message if there is a cryptocashier maintenance', () => {
-        render(
-            <CashierLocked
-                account_status={{
+        const mockRootStore: DeepPartial<TRootStore> = {
```

Comment:
Could we define it outside the test case and just use it in the test case instead of defining it in every test case? I think we can have the general one and just modify the item we want in each test case.

Reply:
@hamid-deriv I personally prefer each test case to be isolated, I understand the concern is code duplication but I don’t think that harms anything in test files.


Let me know if I should change it 🙇🏻🙏🏻

Reply:
Yes, the only concern is duplication. That is just a suggestion. You can ignore it :+1: